gffquant/db/annotation_db.py

```
@@ -180,9 +207,20 @@ def get_features(self, category_id):
     def query_category(self, category_id):
         return self.db_session.query(db.Category).filter(db.Category.id == category_id).one_or_none()
 
+    def get_categories(self):
+        return self.db_session.query(db.Category).all()
+
+    def get_features(self, category=None):
```

Ruff (0.8.0)

Redefinition of unused `get_features` from line 204

(F811)